java-grupa8/homework4/src/main/java/ElectricBatteries.java

```diff
@@ -0,0 +1,2 @@
+public class ElectricBatteries extends Car {
```

Comment:
Verify the OOP questions please:
ElectricBatteries is-a Car?
ElectricBatteries has-a Car?
Car is-a ElectricBatteries?
Car has-a ElectricBatteries?